openedx/adg/lms/courseware_override/tests/test_views.py

```diff
@@ -0,0 +1,130 @@
+"""
+Tests for views.py file of courseware_override
+"""
+from datetime import datetime, timedelta
+
+import mock
+import pytest
+
+from common.djangoapps.student.roles import CourseInstructorRole
+from common.djangoapps.student.tests.factories import UserFactory
+from openedx.adg.lms.applications.tests.factories import MultilingualCourseFactory, MultilingualCourseGroupFactory
+from openedx.adg.lms.courseware_override.views import (
+    get_all_course_instructors,
+    get_all_languages_for_course,
+    get_course_enrollment_count
+)
+from openedx.core.djangoapps.content.course_overviews.tests.factories import CourseOverviewFactory
+
+START_DATE = datetime.today() - timedelta(days=365)
+END_DATE = datetime.today() + timedelta(days=365)
+DUMMY_IMAGE_URLS = {'small': 'small_dummy_url', 'medium': 'large_dummy_url', 'large': 'large_dummy_url'}
+
+
+@pytest.mark.django_db
+@pytest.fixture(name='users')
+def user_fixture(request):
+    """
+    Create test users
+
+    Returns:
+        list: A list of User objects as specified
+    """
+    users = UserFactory.create_batch(request.param)
+    return users
+
+
+@pytest.fixture(scope='function', name='get_course')
+def course_fixture():
+    """
+    Creates a function to create a course with the given language
+
+    Returns:
+        func: A function that allows user to create a course with the specified language
+    """
+    course_list = []
```

Comment:
I don't think this `course_list ` is being used. In one test case where we need a list of courses we are maintaining the list inside the test case. Hence this fixture can be re defined to just include the course creation part